indico/modules/events/layout/util.py

```diff
@@ -349,6 +379,43 @@ def get_css_url(event, force_theme=None, for_preview=False):
         return _build_css_url(layout_settings.get(event, 'theme'))
 
 
+def _build_js_url(theme):
+    if ':' in theme:
+        try:
+            # if the js_path is empty, we don't have a javascript file, so skip
+            if get_plugin_conference_themes()[theme].js_path:
+                path = get_plugin_conference_themes()[theme].js_path
+            else:
+                return None
```

Comment:
```suggestion
            return get_plugin_conference_themes()[theme].js_path
```

Reply:
We can't return immediately, because we're just trying to set the path variable. We use that path variable for the 'actual' return later on below.